cli/outputflags/flags.go

```diff
@@ -22,6 +22,7 @@ import (
 type Flags struct {
 	anyio.WriterOpts
 	DefaultFormat string
+	DefaultPretty bool
```

Comment:
Nit: Call this DefaultZSONPretty since both JSON and ZSON can be pretty?

Reply:
Hmm, good point.  I think the logic is off.  I'll look a little closer and write some tests.

Reply:
Here's the problem...
```
; super compile -f json 'from x'
cannot use -z or -Z with -f
```